### PR TITLE
Trim descriptions to shorten --list display width.

### PR DESCRIPTION
**Pull request by cmnbroad**
Picard part of https://github.com/broadinstitute/gatk/issues/5421.

Before:
screen shot 2018-12-27 at 9 04 18 am

After:
screen shot 2018-12-27 at 9 07 45 am

### CODE REVIEW

**Comment by cmnbroad** [comment]
After some discussion on the GTAK part of this, we decided to add a test to ensure we don't exceed the length, rather than just silently truncating.

File: src/test/java/picard/cmdline/PicardCommandLineTest.java [hunk 1]
```diff
@@ -63,4 +63,19 @@ public void testLaunchAllCommandLineProgramsWithBarclayParser() {
     public void testPrintUsage() {
         Assert.assertEquals(new PicardCommandLine().instanceMain(new String[]{"-h"}), 1);
     }
+
+    @Test
+    public void testCommandlineProgramPropertiesOneLineSummaryLength() {
+        // Find and test each command line tool to ensure that one line
+        // summaries don't exceed the maximum allowable length
+        PicardCommandLine.processAllCommandLinePrograms(
+                Collections.singletonList("picard"),
```
Comment (pshapiro4broad) [inline src/test/java/picard/cmdline/PicardCommandLineTest.java]: It's not too bad, but it does seem wasteful that the tests generates the list of all picard CLPs three times here. If you generated it once in a `BeforeClass` you could use it in all three tests.

Comment (cmnbroad) [reply src/test/java/picard/cmdline/PicardCommandLineTest.java]: Done. Consolidated these into a single `BeforeClass` discovery method.

**Comment by yfarjoun** [comment]
why isn't this merged?

**Pull request merged**